apps/server/src/utils/temporal.ts

```diff
@@ -0,0 +1,40 @@
+import { Maybe } from 'ontime-types';
+import { dayInMs, MILLIS_PER_MINUTE } from 'ontime-utils';
+
+export type Instant = number & { __brand: 'instant' };
```

Comment:
suggestion: i know we used the name instant in our chats. But instants are constructs which carry some baggage we dont want

I would suggest we stick to ontime names, how about
- EpochMs: `startedAt`
- DayMs: `eventNow.timeStart`
- Duration: eg. `elapsed`